contract/artifacts/README.md

```
@@ -34,111 +34,153 @@ This crate deliberately does **not** compile contracts in a build script.
+// Requires: features = ["fetch"]
+use templar_contract_artifacts::{fetch, ArtifactId};
+
+// A specific historical release, for upgrade tests.
+let old = fetch::released_bytes(ArtifactId::UniversalAccount, "0.2.0").await?;
+```
+
+### The cache
+
+Bytes are cached outside the repository so every worktree shares one copy:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)